src/main/java/mike/controller/SaveCancel.java

```diff
@@ -94,10 +94,10 @@ public void actionPerformed(ActionEvent e)
 			      control.getModel().getEntities().get(x).setXLocation(entity.getXLocation());
 			      control.getModel().getEntities().get(x).setYLocation(entity.getYLocation());
 				  for(Field field : entity.getFields()){
-				      control.getModel().createField(entity.getName(), field.getName(), field.getType());
+				      //control.getModel().createField(entity.getName(), field.getName(), field.getType());

```

Comment:
Do these comments still need to be here? There are others in this PR as well, just wondering what they're being saved for.

Reply:
Yeah, they should be there. That commented line is how we create some of the things in the edit box, but since we had to change around how fields and methods are created, it would throw errors if left uncommented.